test/metabase/api/routes_test.clj

```diff
@@ -0,0 +1,42 @@
+(ns metabase.api.routes-test
+  (:require
+   [clojure.test :refer :all]
+   [rewrite-clj.parser]
+   [rewrite-clj.zip :as z]))
+
+(set! *warn-on-reflection* true)
+
+(defn- map-keys [map-loc]
```

Comment:
nit: map-keys commonly means mapping over they keys in a map, took me a second to see that we aren't doing that here. Consider renaming? Maybe find-map-keys